gensim/models/callbacks.py

```diff
@@ -87,6 +93,9 @@ def __init__(self, corpus=None, texts=None, dictionary=None, coherence=None, win
         self.viz_env = viz_env
         self.title = title
 
+    def __str__(self):
```

Comment:
By default, if you have no `method` in child-class, this method will be called from `parent` class -> no need to call `__str__` from each callback explicitly.